buildpack/layermetadata_test.go

```diff
@@ -0,0 +1,47 @@
+package buildpack_test
+
+import (
+	"os"
+	"runtime"
+	"testing"
+
+	"github.com/buildpacks/lifecycle/buildpack"
+	h "github.com/buildpacks/lifecycle/testhelpers"
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+	"github.com/tj/assert"
+)
+
+func TestLayerMetadata(t *testing.T) {
+	if runtime.GOOS != "windows" {
```

Comment:
Why no Windows?

Reply:
Oh. Mostly because I didn't have a Windows machine on the train with me :)  Have changed this.